### ISSUE TITLE
Incompatibility with mysql 5.6

### ISSUE THREAD

**Issue by LoicMahieu** [ArsenyYankovsky/typeorm-aurora-data-api-driver#75]
Hi,

Since the upgrade from `0.2.26` to `0.2.32`, we have an error when trying to insert a `simple-json` field.
See theses logs:
- Query:
```
INSERT INTO `survey_question_user_answer`(`id`, `position`, `questionAnswerId`, `questionId`, `userSubmissionId`) VALUES (DEFAULT, :param_0, DEFAULT, :param_1, :param_2) -- PARAMETERS: [{"name":"param_0","value":"{\"top\":18.149767019916705,\"left\":43.66928614606803}","cast":"JSON"},{"name":"param_1","value":34},{"name":"param_2","value":1490}]
```
- Error:
```
BadRequestException: Database error code: 1064. Message: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'JSON), DEFAULT, 34, 1490)' at line 1
```

I think it's related to commit https://github.com/ArsenyYankovsky/typeorm-aurora-data-api-driver/commit/22729297fa20815acb4351c2bee2d4baf9c10c58 where the value transformed before being persisted.

Is there a word around to rollback to the previous behavior ?

Thanks a lot!

**Comment by ArsenyYankovsky** [comment]
Hi, setting `formatOptions.castParameters` to `false` should revert to the old behavior. I'll add tests for the `simple-json` type for MySQL when I get a chance and provide a proper fix. Thanks for reporting.

**Comment by ArsenyYankovsky** [comment]
@LoicMahieu Hey, I released a new version of the driver (2.0.6) with a fix for the `simple-json` columns.

**Comment by LoicMahieu** [comment]
Hi @ArsenyYankovsky 
Thank you a lot for your responses! And sorry to be so late for mine, I was busy for other things ;)

Great! I updated
```
 typeorm-aurora-data-api-driver  2.0.5  →  2.1.0
```

And I works well. Thanks !
